news/changelog-empty.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* No news added: Add CHANGELOG.rst.
```

Comment:
for news items that don't need to be compiiled we can smiply run 

`package add news -n -m "Add CHANGELOG.rst."`